src/ingestor.ts

```diff
@@ -154,7 +154,9 @@ export default async function ingestor(req) {
   try {
     context = await writer[type].verify(legacyBody);
   } catch (e) {
-    capture(e);
+    if (typeof e !== 'string') {
+      capture(e);
+    }
```

Comment:
maybe better to handle this condition in capture itself? so we don't have to worry about adding it in the future? 

we may forgot adding it

Reply:
This is not systematic